src/slideshow.ts

```diff
@@ -0,0 +1,51 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
HTMX?

Reply:
i am not using HTMX bs it is slideshow for photo